src/applications/mhv/secure-messaging/containers/FolderListView.jsx

```diff
@@ -132,7 +132,7 @@ const FolderListView = props => {
 
   return (
     <div className="vads-l-grid-container vads-u-padding--0">
-      <div className="main-content">
+      <div className="main-content folder-list-view">

```

Comment:
same as above for `secure-messaging` class. We could always use `data-testid `attribute as well if we are not relying on this class declaration for styling